### PR TITLE
Bump to version 3.23.1

### PR DESCRIPTION
**Pull request by chambbj**

### CODE REVIEW

**Comment by chambbj** [comment]
@conda-forge-admin, please rerender

**Comment by chambbj** [comment]
@conda-forge-admin, please rerender

**Comment by chambbj** [comment]
@conda-forge/sqlite Spent more time on this today than I'd like to admit, but I think this PR is probably in pretty good shape finally. The intent all along was simply to bump the version to 3.23.1 and to modernize the meta.yaml.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,46 +1,45 @@
...
-    - conda inspect linkages -p $PREFIX $PKG_NAME  # [not win]
-    - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
+    - if not exist %PREFIX%\\Library\bin\sqlite3.dll exit 1    # [win]
+    - if not exist %PREFIX%\\Library\lib\sqlite3.lib exit 1    # [win]
+    - test -f $PREFIX/lib/libsqlite3.dylib                     # [osx]
+    - test -f $PREFIX/lib/libsqlite3.so                        # [linux]
+    - if not exist %PREFIX%\\Library\include\sqlite3.h exit 1  # [win]
+    - test -f $PREFIX/include/sqlite3.h                        # [not win]
+    - conda inspect linkages -p $PREFIX $PKG_NAME              # [not win]
+    - conda inspect objects -p $PREFIX $PKG_NAME               # [osx]
```
Comment (jakirkham) [inline recipe/meta.yaml]: We should drop these last two lines. `conda inspect` is on its way out. Also `conda-build` already provides this information much more clearly during the build.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,46 +1,45 @@
-{% set version="3.22.0" %}
-{% set ver = version.replace(".", "") + "000" %}
+{% set name = "sqlite" %}
+{% set version = "3.23.1" %}
+{% set ver_major = version.split(".")[0] %}
+{% set ver_minor = version.split(".")[1] %}
+{% set ver_patch = version.split(".")[2] %}
+{% set ver = "".join([ver_major, ver_minor.ljust(3, "0"), ver_patch.ljust(3, "0")]) %}
```
Comment (jakirkham) [inline recipe/meta.yaml]: Thanks for the improvements here. Maybe we should call this `ver_url` for clarity.

**Comment by jakirkham** [comment]
Thanks for working on this, @chambbj. See several nice improvements in here. Made a couple minor comments. Once addressed, happy to merge.

**Comment by chambbj** [comment]
I made the requested changes @jakirkham. This should be good to go, followed by #28.

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

**Comment by jakirkham** [comment]
Thanks @chambbj :)

**Pull request merged**